frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/StartHandler.java

```diff
@@ -160,14 +159,9 @@ public APIGatewayProxyResponseEvent handleRequest(
                 LOG.info("Subject saved to ClientSession for DocCheckingAppUser");
             }
 
-            StartResponse startResponse;
-            if (configurationService.isExtendedFeatureFlagsEnabled()) {
-                Features features = startService.getSessionFeatures();
-                LOG.info("Extended feature flags enabled: {}", features);
-                startResponse = new StartResponse(userStartInfo, clientStartInfo, features);
-            } else {
-                startResponse = new StartResponse(userStartInfo, clientStartInfo);
-            }
+            StartResponse startResponse =
+                    new StartResponse(
+                            userStartInfo, clientStartInfo, startService.getSessionFeatures());
```

Comment:
Re @BeckaL's comment, looks like the 'else' part of the if statement should have been kept rather than this bit.  We would like to remove the extended feature flags facility completely, including Features and the getSessionFeatures method.

Reply:
cleaned up, originally wasn't sure if we wanted it for future enhancements.